src/main/resources/validationrules/sscs/sscs-judicial-org-role-mapping.drl

```diff
@@ -165,9 +278,12 @@ then
    attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("SSCS"));
    attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($joh.getPrimaryLocation()));
    attribute.put("contractType", JacksonUtils.convertObjectIntoJsonNode("Fee-Paid"));
-   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("hearing_work,decision_making_work,routine_work,"
-                                                                      + "priority"));
+   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("pre_hearing_work,hearing_work,post_hearing_work,"
+                                                                        + "decision_making_work,routine_work,priority"));
    attribute.put("region", JacksonUtils.convertObjectIntoJsonNode($joh.getRegionId()));
+   if ($joh.getTicketCodes().contains("368")) {
```

Comment:
Can we null protect this by adding a != null check in front of this?

Reply:
added in patch branch. see 2f575c6